README.rst

```diff
@@ -572,28 +573,28 @@ Here is an example of ``.cartridge.yml``:
     script: my-init.lua
     stateboard: true
 
-**Note:** the config of the `standard application template <Creating an application from template_>`_ initially has the ``stateboard: true`` parameter.
+**Note:** the config of the `standard application template <Creating an application from template_>`_
+initially has the ``stateboard`` parameter set to ``true``.
 
-.. // Please, update the doc in cli/commands on updating this section
+.. // Please update the doc in cli/commands on updating this section
 
-*********
+********
 ``stop``
-*********
+********
 
-To stop one or more running instances, say:
+To stop one or more running instances, execute:
 
 .. code-block:: bash
 
     cartridge stop [INSTANCE_NAME...] [flags]
 
-By default, SIGTERM is sent to instances.
+By default, the instances receive a SIGTERM.
 
-The following options (``[flags]``) are supported:
+Supported options (``[flags]``):
 
-* ``-f, --force`` indicates if instance(s) stop should be forced (sends SIGKILL).
+* ``-f, --force`` allows force-stopping the instance(s) by a SIGKILL.
```

Comment:
I think it's 'with', because SIGKILL is just a number, not some program or tool.

Reply:
OK